.github/workflows/deploy-extension-to-marketplace.yml

```diff
@@ -0,0 +1,47 @@
+name: Docker Build and Push to AWS Marketplace
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_tag:
+        description: 'Tag for the Docker image'
+        required: true
+        default: '' # Pass the value like 1.0.0
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+    paths:
+      - 'Dockerfile'
+
+jobs:
+  build:
+      if: github.actor == 'dependabot' || github.event_name == 'workflow_dispatch'
```

Comment:
But having dependabot here also means that everytime we merge a Java dependency we will release it to docker, right? Maybe we need additional checks?

Reply:
the below condition will take precedence over the `if:` 
`types:
      - closed
    paths:
      - 'Dockerfile'`